variables.tf

```diff
@@ -340,6 +340,17 @@ variable "log_analysis_targets" {
   sensitive   = true
 }
 
+# Cloud Logs Targets
+variable "at_cloud_logs_targets" {
+  type = list(object({
+    instance_id   = string
+    target_region = optional(string)
+    target_name   = string
+  }))
+  default     = []
+  description = "List of cloud logs target to be created"
```

Comment:
It's a list, so should say "targets" instead of "target"

Reply:
Also Cloud Logs is a service, so they should probably start with caps. You used the correct description in the submodule, but not here. Lets try keep them, consistent.